code-challenges.js

```diff
@@ -10,23 +10,86 @@ var people = [
 ]
 // Expected output example: "Ford Prefect is a hitchhiker." "Zaphod Beeblebrox is a president of the galaxy." "Arthus Dent is a radio employee."
 
+// My psuedocode: (1) Map through the array to capitalize name. (2) Concatenate name with occupation. 
+const createSen = (array) => {
+  let capName = array.map(value => {
+    value.name.charAt(0).toUpperCase() + name.slice(1)
+  })
+  // encountered an error that needs more time for a solution but I took the time to understand the class answer. 
+}
+console.log(createSen(people))
```

Comment:
Ah! So close! I'd love to see this finished up, I think you can get past a few of the errors by having a return of capName and examining a bit closer to how you're calling .slice on name. You'll need some more logic here to produce what you need to, but you're getting close!